src/main/java/com/strandls/cca/service/impl/CCATemplateServiceImpl.java

```diff
@@ -327,4 +328,34 @@ public Activity addComment(HttpServletRequest request, Long userId, CommentLoggi
 		return activity;
 	}
 
+	
+	
+	@Override
+	public List<String> getFieldIds(String shortName, String language) {
+		if (shortName == null || "".equals(shortName))
```

Comment:
For empty string comparison, do it like this in java `shortName.isEmpty() `instead of using equals with ""

Reply:
done